scripts/scripts/local_s3_server.py

```
@@ -0,0 +1,126 @@
+    )
+    return parser
+
+
+def main(directory: str, host: str, port: int) -> None:
+    root = Path(directory).resolve()
+    if not root.exists() or not root.is_dir():
+        raise SystemExit(f"Directory does not exist or is not a directory: {root}")
+
+    handler = lambda *args, **kwargs: CORSRequestHandler(
+        *args, directory=str(root), **kwargs
+    )
```

Ruff (0.12.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `handler` as a `def`

(E731)